pvl/lexer.py

```diff
@@ -343,7 +293,7 @@ def lex_continue(char: str, next_char: str, lexeme: str,
     # Some datetimes can have trailing numeric tz offsets,
     # if the decoder allows it, this means there could be
     # a '+' that splits the lexeme that we don't want.
-    if (next_char in g.numeric_start_chars and token.is_datetime()):
+    if next_char in g.numeric_start_chars and token.is_datetime():
```

Comment:
you could actually replace the if and the two returns to just: `return next_char in g.numeric_start_chars and token.is_datetime()` but it is sort of a style choice

Reply:
You could.  This part of the code has a list of these if statements, and then the function concludes with a final `return False` which allows more of these conditions to be slipped in, if they are needed in the future.